radio/src/Makefile

```diff
@@ -228,7 +228,11 @@ PXX = NO
 
 # DSM2 (Spektrum) protocol
 # Values = NO, SERIAL, PPM
-DSM2 = NO
+DSM2 = PPM
+
+# Multimodule support
+# (http://www.rcgroups.com/forums/showthread.php?t=2165676)
+MULTIMODULE = NO
```

Comment:
You will need to add the option in the radio/src/CMakeLists.txt (in the 'next' branch)